src/utils/PlottingUtils.jl

```diff
@@ -0,0 +1,145 @@
+module PlottingUtils
+
+using Plots
+using Cloudy.ParticleDistributions
+
+export plot_moments!, plot_spectra!
+
+function plot_moments!(sol, p; plt_title="test_moments")
+    Plots.gr()
+    time = sol.t
+    moment_plots = []
+    if p.Ndist == 1
+        for i in 1:p.Nmom
+            ploti = plot(time,
+                    vcat(sol.u...)[:,i],
+                    linewidth=2,
+                    xaxis="time [s]",
+                    yaxis="M"*string(i-1),
+                    label="M_{"*string(i-1)*",1}",
+                    xlims=(0, maximum(time)),
+                    ylims=(0, 1.2*maximum(vcat(sol.u...)[:,i]))
+                )
+            push!(moment_plots, 
+                ploti
+            )
+        end
+    else
+        plotu = reshape(vcat(sol.u...), (p.Ndist,length(time),p.Nmom))
+        for i in 1:p.Nmom
+            ploti = plot(time,
+                    plotu[1,:,i],
+                    linewidth=2,
+                    xaxis="time [s]",
+                    yaxis="M"*string(i-1),
+                    label="M_{"*string(i-1)*",1}",
+                    xlims=(0, maximum(time)),
+                    ylims=(0, 1.2*maximum(plotu[1,:,i]))
+                )
+            for j in 2:p.Ndist
+                plot!(ploti,
+                    time,
+                    plotu[j,:,i],
+                    linewidth=2,
+                    label="M_{"*string(i-1)*","*string(j)*"}",)
+            end
+            plot!(ploti,
+                time,
+                vcat(sum(plotu[:,:,i], dims=1)...),
+                linestyle=:dash,
+                linecolor=:black,
+                label="M_"*string(i-1),
+                linewidth=2
+            )
+            push!(moment_plots, 
+            ploti
+        )
+        end
+    end
+    plot(moment_plots...)
+    savefig("examples/"*plt_title*".png")
+end
+
+function plot_spectra!(sol, p; plt_title="test_spectra", logxrange=(0, 8))
```

Comment:
It would be nice to add a small doc above this function explaining what it plots and what the inputs are.